test/api/fixtures.h

```diff
@@ -51,6 +51,9 @@ struct Fixtures {
 
     std::string input_jpg_overlay_layer_2{dir + "/alpha-layer-2-ink.jpg"};
 
+    // https://images.unsplash.com/photo-1586015910630-34cc709e5f91
+    std::string input_jpg_saturate{dir + "/photo-1586015910630-34cc709e5f91.png"}
+
```

Comment:
Could you not use [`input_jpg`](https://raw.githubusercontent.com/weserv/images/5.x/test/api/fixtures/2569067123_aca715a2ee_o.jpg) instead? I don't see the point of including another test image.